llumnix/output_queue/output_queue_client_base.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2024, Alibaba Group;
```

Comment:
queue_client_base, queue_server_base is enough. Also, for directory, output_queue -> queue, because although we use queue to put/get request output token, but the queue and its interface is general, not specific to request output.

Reply:
get